sahi/predict.py

```diff
@@ -460,14 +472,97 @@ def predict(
     # iterate over source images
     durations_in_seconds["prediction"] = 0
     durations_in_seconds["slice"] = 0
-    for ind, image_path in enumerate(tqdm(image_path_list, "Performing inference on images")):
+
+    # Input video
+    if export_video:
+        if video_path is not None:
+            #get video name with extension
+            image_base = os.path.basename(video_path)
+        else:
+            print("\nVideo path is none. Enter (--video_path 'path' --export_video) in terminal!\n")
```

Comment:
Shouldn't we raise exception (`ValueError`) here ? If we rely on L484, then just simply remove the `else` block.

**A side note:** `"\n"` are redundant here I think, you do not need these in Python as it will by default print with a new line, and end with a newline.

Reply:
I didn't think this, your opinion is better than